src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpMultilineCheck.java

```diff
@@ -104,6 +104,32 @@
  * greedy will allow multiple violations in the example to be found in the same file.
  * </p>
  *
+ * <p>
+ * To configure the check to restrict an empty file:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;RegexpMultiline&quot;&gt;
+ *     &lt;property name=&quot;format&quot; value=&quot;^\s*$&quot; /&gt;
+ *     &lt;property name=&quot;matchAcrossLines&quot; value=&quot;true&quot; /&gt;
+ *     &lt;property name=&quot;message&quot; value=&quot;Empty file is not allowed&quot; /&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>
+ * Example of violation from the above config:
+ * </p>
+ * <pre>
+ * /var/tmp$ cat -n Test.java
+ * 1
+ * 2
+ * 3
+ * 4
+ * </pre>
+ * <p>Result:</p>
+ * <pre>
+ * /var/tmp$ java -jar checkstyle-8.31-all.jar -c config.xml Test.java
```

Comment:
Please remove only this line, most of our users use checkstyle through plugins.

Reply:
Done